sentry_sdk/scope.py

```diff
@@ -196,10 +196,23 @@ def _create_new_propagation_context(self):
             "dynamic_sampling_context": None,
         }
 
+    def set_new_propagation_context(self):
+        # type: () -> None
+        """
+        Creates a new propagation context and sets it as `_propagation_context`. Overwriting existing one.
+        """
+        logger.debug(
+            "[Tracing] Create new propagation context: %s",
+            self._propagation_context,
+        )
```

Comment:
This should probably be logged after we've set the new propagation context on the next line

Reply:
done!